pallets/gear/src/weights.rs

```diff
@@ -19,12 +19,12 @@
 //! Autogenerated weights for pallet_gear
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2022-09-17, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
-//! HOSTNAME: `epyc-runners-node.hetzner`, CPU: `AMD EPYC 7502P 32-Core Processor`
+//! DATE: 2022-11-05, STEPS: `51`, REPEAT: 200, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! HOSTNAME: `ip-172-31-20-193.us-west-1.compute.internal`, CPU: `Intel(R) Xeon(R) Platinum 8375C CPU @ 2.90GHz`
```

Comment:
Is it ok to see here ip of the machine?

Reply:
Hmm, may be it's not ok. Ok,  I will remove this

Reply:
Removed it